vizstorm-GIF-Florence-2018.Rproj

```diff
@@ -11,7 +11,3 @@ Encoding: UTF-8
 
 RnwWeave: knitr
 LaTeX: pdfLaTeX
-
-AutoAppendNewline: Yes
-StripTrailingWhitespace: Yes
```

Comment:
oh, this is why the trailing whitespace is not being stripped

Reply:
not a huge deal, but i do like how setting this to Yes reduces the number of whitespace changes we make in PRs

Reply:
oh, didn't realize this. this must be the default. I started a new project w/ an empty dir, and then pointed the remotes to github to pull and build from there. I probably should've kept the original .proj that got pulled down.